app/src/pages/Landing.tsx

```diff
@@ -23,10 +32,31 @@ const BottomRightContainer = styled.div`
 `;
 
 export function LandingPage(): JSX.Element {
+  const connected = useSelector(
+    (state: RootState) => state.server.servers[state.server.selectedServer]
+  );
+
+  return <AppContainer>{connected ? <LandingContent /> : <ConnectingContent />}</AppContainer>;
+}
+
+function ConnectingContent(): JSX.Element {
+  return (
+    <>
+      <TitleBar />
+      <MainCenterColumn>
+        <Heading>Connecting to server...</Heading>
```

Comment:
I think the wording here is not really helpfull. I would rather prefer something like starting app up. Server sounds like we are leaking private info somewhere